app/routes.js

```diff
@@ -1,13 +1,18 @@
 // @flow
 import React from 'react';
-import { Route } from 'react-router';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Switch, Route } from 'react-router';
 import App from './containers/App';
 import HomePage from './containers/HomePage';
 import CounterPage from './containers/CounterPage';
 
-export default (
-  <Route path="/" component={App}>
-    <Route path="/" component={HomePage} />
-    <Route path="/counter" component={CounterPage} />
-  </Route>
+export default () => (
+  <Router>
+    <App>
```

Comment:
The line in question was https://github.com/thisconnect/electron-react-boilerplate/blob/4950ad98afcf25a2ae7734d5406c76abc345ea8e/app/containers/App.js#L7

I'm not exactly sure where to look for react flow types, especially children. My personal opinion is that tools like flow should help the developer. However I find it quite hard to google this particular lint warning and am not sure where to look. flow, flow-type something lint?
No mention about children https://flow.org/en/docs/frameworks/react/

This issue came up in my search but didn't help https://github.com/facebook/flow/issues/1355

This seems to be the type definition for react https://github.com/facebook/flow/blob/master/lib/react.js#L157

My last commit does lint now and it looks right to me https://github.com/thisconnect/electron-react-boilerplate/commit/ced774dc86241004ae0e80a8aac8f2e85ee1387c